# Update to give us permission to modify port routing.

diff --git a/node_modules/@wordpress/env/lib/commands/run.js b/node_modules/@wordpress/env/lib/commands/run.js
index def29b6..fc127ce 100644
--- a/node_modules/@wordpress/env/lib/commands/run.js
+++ b/node_modules/@wordpress/env/lib/commands/run.js
@@ -85,6 +85,7 @@ function spawnCommandDirectly( config, container, command, envCwd, spinner ) {
 		envCwd,
 		'--user',
 		hostUser.fullUser,
+		'--privileged'
 	];

 	if ( ! process.stdout.isTTY ) {
